src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -501,16 +501,9 @@ CHIP_ERROR AccessControlAttribute::WriteExtension(AttributeValueDecoder & aDecod
 
 AccessControlAttribute gAttribute;
 
-AccessControl gAccessControl(Examples::GetAccessControlDelegate());
```

Comment:
Can the include be removed now?
`#include <access/examples/ExampleAccessControlDelegate.h>`